content/rendering-with-batch/aws-fis/fis-start-job.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Starting a new render job"
+date: 2021-09-06T08:51:33Z
+weight: 154
+---
+
+This section of the lab will start a new rendering job. This second job will be intentionally interrupted using AWS FIS.
```

Comment:
Avoid calling this a lab, that terminology has not been used anywhere in the other sections. Also, don't say that the section does something, is the user who will perform some actions in the section to accomplish something. For instance, rephrase to "In this section you are going to start a new rendering job that will be intentionally interrupted by AWS FIS using the experiment that you have created in the previous section."